todo/ios/Podfile.lock

```diff
@@ -0,0 +1,126 @@
+PODS:
```

Comment:
this file should not be committed, it will be generated by users when they run `pod install`

Reply:
Done. I thought generally dependency lockfiles should be committed though, and represent a "known working" state?

Reply:
Actually you're right, upon further examination. Anything in the `Pods` directory does not need to be checked in however